gmake test reports that poll() using WASI fails on latest Solaris 11.4:

=== release test-wasi ===
Path: wasi/test-wasi
...
executing poll
(node:7530) ExperimentalWarning: WASI is an experimental feature. This feature could change at any time
fd_prestat_get(3, 69896)
fd_prestat_dir_name(3, 131088, 9)
fd_prestat_get(4, 69896)
fd_prestat_dir_name(4, 131152, 5)
fd_prestat_get(5, 69896)
environ_sizes_get(69900, 69896)
environ_get(132688, 131168)
clock_time_get(0, 1000000000, 69752)
poll_oneoff(69696, 69656, 1, 69692)
clock_time_get(0, 1000000000, 69752)
clock_time_get(0, 1000000000, 69752)
poll_oneoff(69584, 69552, 1, 69756)
clock_time_get(0, 1000000000, 69752)
poll_oneoff(69488, 69424, 2, 69756)
poll_oneoff(69488, 69424, 2, 69756)
poll_oneoff(69584, 69520, 2, 69756)
fd_write(2, 69504, 2, 69500)
Assertion failed: fds[0].revents == (POLLHUP | POLLIN) (c/poll.c: main: 70)

--- node-22.22.0/test/wasi/wasi.status
+++ node-22.22.0/test/wasi/wasi.status
@@ -3,6 +3,9 @@ prefix wasi
 # To mark a test as flaky, list the test name in the appropriate section
 # below, without ".js", followed by ": PASS,FLAKY". Example:
 # sample-test                        : PASS,FLAKY
+# Solaris
+[$system==solaris]
+test-wasi-poll: SKIP
 
 [true] # This section applies to all platforms
 
